modules/apps/portal-template/portal-template-velocity/src/main/java/com/liferay/portal/template/velocity/internal/helper/VelocityTemplateContextHelper.java

```diff
@@ -152,11 +151,21 @@ public void prepare(
 
 	@Activate
 	@Modified
-	protected void activate(Map<String, Object> properties) {
+	protected void activate(
+		BundleContext bundleContext, Map<String, Object> properties) {
+
+		_serviceTrackerList = ServiceTrackerListFactory.open(
```

Comment:
@W2jx For this one, you can just re-create the configuration object in your modified method.